service_container/expression_language.rst

```diff
@@ -125,3 +125,44 @@ via a ``container`` variable. Here's another example:
 
 Expressions can be used in ``arguments``, ``properties``, as arguments with
 ``configurator`` and as arguments to ``calls`` (method calls).
+
+You can also use expressions as factories to create services based on an expression:
```

Comment:
What about shortening this to `You can also use expressions as service factories:`